lib/config.js

```diff
@@ -10,6 +10,20 @@ require('bedrock-permission');
 
 config.identity = {};
 
+/**
+ * An extendable list of accepted fields for each identity model
+ */
+config.identity.fields = [
+  'identity.description',
+  'identity.image',
+  'identity.label',
+  'identity.sysGravatarType',
+  'identity.sysImageType',
+  'identity.sysPublic',
+  'identity.sysSigningKey',
+  'identity.url'
+];
```

Comment:
I think we should make this a map w/`true` values because it's easier to switch them off/on when configuring.